appstudio-utils/util-scripts/lib/fetch/cluster.sh

```diff
@@ -21,9 +21,38 @@ _default-policy-config() {
   echo '{"non_blocking_checks":["not_useful"]}'
 }
 
+# Splits the given string on '/' and returns the second part
+cr-name() {
+  echo "${1//*\//}"
+}
+
+cr-namespace-argument() {
+  if [[ "$1" != */* ]]; then
+    return
+  fi
+
+  local namespace="${1//\/*/}"
+  if [[ -n "${namespace}" ]]; then
+    echo "-n ${namespace}"
+  fi
+}
+
 # Placeholder. In future it will likely come from
 # an EnterpriseContractPolicy crd, see HACBS-244
 save-policy-config() {
+  local args=($(cr-namespace-argument "$1") $(cr-name "$1"))
+  if ! non_blocking_data=$(kubectl get enterprisecontractpolicies.appstudio.redhat.com "${args[*]}" -o jsonpath='{.spec.exceptions.nonBlocking}'); then
+    echo "ERROR: unable to find the ec-policy EnterpriseContractPolicy in namespace $(kubectl config view --minify -o jsonpath='{..namespace}')" 1>&2
```

Comment:
The `kubectl config view` command may return an empty string if the default namespace is selected. At least that's what [stackoverflow](https://stackoverflow.com/questions/55853977/how-to-get-the-current-namespace-of-current-context-using-kubectl) says. Maybe it's not a big deal for us. An alternative is to query for the namespace of any of the ServiceAccounts.

Reply:
I'm okay with it returning "" the message will be `"... in namespace"` so it'll at least prompt someone to set the namespace. Reworked this a bit it didn't take into consideration if custom namespace was specified.